spec/rachets/medical_history_boolean_fields_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "spec_helper"
+
+RSpec.describe "Rachet down usage of boolean fields in medical history" do
+  it "prevents new usages of prior_heart_attack_boolean" do
+    expected_usages = 1
```

Comment:
What's the one usage we can't remove yet? `structure.sql`?

Can we axe the field completely?

Reply:
Yeah. Axing this field from our schema will require an API version bump, hence we've not done that here.